react/util/ui-utils.js

```diff
@@ -144,6 +146,14 @@ export const notify = function (message) {
         : Alert.alert(message);
 };
 
+export const toClipboard = function (value, notification = true) {

```

Comment:
Maybe it would be cool to use more fine-grained config
```suggestion
export const toClipboard = function (value, { notify = true, vibrate = true } = {}) {

```

Reply:
notify is already declared and can't be used as arg, using `notification` instead